notebooks/datetime.ipynb

```diff
@@ -256,7 +257,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 19,
+   "execution_count": 55,
```

Comment:
In the future, I would ignore those diffs when committing changes.

Reply:
In the future, this repository won't contain any jupyter notebooks. They really just are notebooks for me. My interactive shell for testing code snippets :-)